src/coreclr/pal/src/safecrt/tcsncat_s.inl

```diff
@@ -3,15 +3,11 @@
 
 /***
 *tcsncat_s.inl - general implementation of _tcscpy_s
-*
-
 *
 *Purpose:
 *       This file contains the general algorithm for strncat_s and its variants.
-*
 ****/
 
-_FUNC_PROLOGUE
```

Comment:
Are we interested to tweak the safecrt functions? I'd expect them to be converted/removed in total.

Reply:
This cleans up dead code, as `_FUNC_PROLOGUE` expands to nothing.